app/models/user.rb

```diff
@@ -5,4 +5,12 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable
   has_many :questions
   has_many :answers
+
+    # validates :alias, presence: true
+  validates :alias, presence: true, length: { minimum: 5 }, :uniqueness => {:message => 'already taken, alias must be unique'}
```

Comment:
Optional style suggestion: you can replace the old format (called a "hash rocket"):
```ruby
:uniqueness => {:message =>
```
with:
```ruby
uniqueness: { message: 'already...
```